### ISSUE TITLE
Expo EAS compilation error (iOS only)

### ISSUE THREAD

**Issue by VolkerLieber** [henninghall/react-native-date-picker#437]
**Describe the bug**
Since updating to the latest version (4.1.3) I can't compile for iOS anymore. Android still compiles without error.

```
(node_modules/react-native-date-picker/ios/RNDatePicker/DatePicker.m:15:29)

  13 | @interface DatePicker ()
  14 | 
> 15 | @property (nonatomic, copy) RCTBubblingEventBlock onChange;
     |                             ^ declaration of 'RCTBubblingEventBlock' must be imported from module 'React.RCTComponent' before it is required
  16 | @property (nonatomic, assign) NSInteger reactMinuteInterval;
  17 | 
  18 | @end 
```

**Expected behavior**
Should compile.

**To Reproduce**
Create Project 
Install react-native-date-picker 4.1.3
eas build --profile development

**Smartphone:**
 - OS: iOS
 - React Native version 0.64.12
 - react-native-date-picker version 4.1.3

**Comment by fobos531** [comment]
The same happens with me with every version from 4.0.0 to 4.1.3 and using expo SDK 44. React native version is 0.64.3

**Comment by vamshi9666** [comment]
same for me with expo@44

**Comment by haibert** [comment]
Hi guys, any updates on this? just upgraded to 44 and my ios build is crashing with the same error..

**Comment by haibert** [comment]
Guys this isnt the expo used package... expo uses this one https://github.com/react-native-datetimepicker/datetimepicker

**Comment by haibert** [comment]
I can confirm that this diff fixed the issue! I did a patch using patch-package. Thank you so much Gustavo [Fix missing import related to issue](https://github.com/gustavoggs/react-native-date-picker/commit/5abb67e164270501c293c61a6fe0adda2c13cb84)

**Comment by gustavoggs** [comment]
@henninghall update the PR to follow what expo suggest, let me know if I need to change anything or if it is good to be released.

**Comment by devcoderbuzz** [comment]
Any one merging this back to main  so that  it works for us?

**Comment by henninghall** [comment]
This doesn't seem to be an issue in latest Expo version. 

If anyone is still having this issue, try Expo 44.0.6 or later. If it doesn't solve it for you, please write a note in this thread.

**Comment by chiubaka** [comment]
^I'm in the middle of upgrading to Expo 44 and am encountering this. Was not encountering this before attempting the upgrade.

**Comment by chiubaka** [comment]
Yeah, I just pinned `"expo": "^44.0.6"` in `package.json` and verified in my `yarn.lock` that I'm getting 44.0.6. Still appear to be building with this issue. Going to try the patch suggested above.

**Comment by chiubaka** [comment]
Applying the patch suggested by @haibert resolved the build issue for me on expo 44.0.6

**Comment by akinzalowevidation** [comment]
> This doesn't seem to be an issue in latest Expo version.
> 
> If anyone is still having this issue, try Expo 44.0.6 or later. If it doesn't solve it for you, please write a note in this thread.

I am using `expo@44.0.6` and `react-native-date-picker@4.2.0` and I am running in this issue.

```
› Compiling react-native-date-picker Pods/react-native-date-picker » DatePicker.m

❌  (node_modules/react-native-date-picker/ios/RNDatePicker/DatePicker.m:15:29)

  13 | @interface DatePicker ()
  14 | 
> 15 | @property (nonatomic, copy) RCTBubblingEventBlock onChange;
     |                             ^ declaration of 'RCTBubblingEventBlock' must be imported from module 'React.RCTComponent' before it is required
  16 | @property (nonatomic, assign) NSInteger reactMinuteInterval;
  17 | 
  18 | @end
```

**Comment by KROSF** [comment]
@akinzalowevidation make sure you have this line on your `Podfile`

```rb
  use_expo_modules!
  post_integrate do |installer|
    begin
      expo_patch_react_imports!(installer)
    rescue => e
      Pod::UI.warn e
    end
  end
```

**Comment by akinzalowevidation** [comment]
@KROSF Thanks for the reply. Unfortunately I already had that in my `Podfile`.

**Comment by wassgha** [comment]
@henninghall can confirm that the issue still appears. The patch provided [here](https://github.com/gustavoggs/react-native-date-picker/commit/5abb67e164270501c293c61a6fe0adda2c13cb84) fixes it, please include it in the next version.

**Comment by the-kenneth** [comment]
Im having this issue on expo 44.0.6 as well, is there a current work around?
